styles.css

```diff
@@ -33,7 +45,11 @@ img{
 }
 
 @media(max-width:600px){
```

Comment:
question was asking to reorder the elements when on smaller screens, how could we do that with flexbox?